packages/components/src/font-size-picker/index.js

```diff
@@ -52,14 +53,17 @@ function getSelectOptions( optionsArray, disableCustomFontSizes ) {
 	} ) );
 }
 
-export default function FontSizePicker( {
-	fallbackFontSize,
-	fontSizes = [],
-	disableCustomFontSizes = false,
-	onChange,
-	value,
-	withSlider = false,
-} ) {
+function FontSizePicker(
+	{
+		fallbackFontSize,
+		fontSizes = [],
+		disableCustomFontSizes = false,
+		onChange,
+		value,
+		withSlider = false,
+	},
+	ref
```

Comment:
it's probably the place where the error I shared is triggered. Maybe it requires a fallback to `createRef()`?